src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/Context/FormattingContext.IndentationData.cs

```diff
@@ -72,6 +96,33 @@ public SyntaxToken EndToken
             {
                 get { return this.Operation.EndToken; }
             }
+
+            public override int Indentation => _lazyIndentationDelta.Value + _baseIndentationGetter();
+
+            protected override IndentationData WithTextSpanCore(TextSpan span)
+            {
+                return new RelativeIndentationData(InseparableRegionSpan.Start, span, Operation, _lazyIndentationDelta, _baseIndentationGetter);
+            }
+        }
+
+        private sealed class AdjustedIndentationData : IndentationData
+        {
+            public AdjustedIndentationData(TextSpan textSpan, IndentationData baseIndentationData, int adjustment)
+                : base(textSpan)
+            {
+                BaseIndentationData = baseIndentationData;
+                Adjustment = adjustment;
+            }
+
+            public IndentationData BaseIndentationData { get; }
+            public int Adjustment { get; }
```

Comment:
i wouldn't mind some docs on this.  the relation of base-indentation, adjustment, and text span aren't super clear.

Reply:
I'm not sure what the text span is for, but I can add docs to explain the other two. 😄